composer.json

```diff
@@ -14,10 +14,13 @@
     }],
     "require": {
         "php": ">=5.3.0",
-        "symfony/framework-bundle" : ">=2.0"
+        "symfony/framework-bundle" : ">=2.0",
+        "doctrine/cache": "~1.3",
+        "guzzlehttp/guzzle": ">=4.0"
```

Comment:
We shouldnt rely on an external library, we havent yet, and should be possible to keep it that way.